src/test/java/org/folio/rest/impl/OaiPmhImplTest.java

```diff
@@ -2196,9 +2199,10 @@ void shouldRespondWithInternalServerError_whenGetOaiRecordsMarc21WithHoldingsWit
       .statusCode(500)
       .extract()
       .asString();
-    String expectedMessage = format(EXPECTED_ERROR_MESSAGE, "source-record-storage");
+    String expectedMessage = "SRS didn't respond with expected status code after 1 attempts. Canceling further request processing.";
```

Comment:
Unit test for re-requesting srs was written before this PR
It is called shouldRerequestSrsWhenItRespondedWith500_whenGetOaiRecordsMarc21WithHoldingsWithErrorResponseAndThenSuccessResponseFromSrs